pom.xml

```diff
@@ -83,7 +83,7 @@
     <google.cloud.pubsub.version>1.108.1</google.cloud.pubsub.version>
     <google.cloud.spanner.version>6.10.1</google.cloud.spanner.version>
     <google.cloud.speech.version>1.24.7</google.cloud.speech.version>
-    <google.cloud.storage.version>1.118.0</google.cloud.storage.version>
+    <google.cloud.storage.version>2.3.0</google.cloud.storage.version>
```

Comment:
Can we upgrade to the latest gcloud version `2.27.0`?

Reply:
We should try to upgrade to the latest version available.

Reply:
https://github.com/data-integrations/google-cloud/pull/1307. Pls have a look

Reply:
Why a separate PR?